### PR TITLE
feat: add basic login form structure

### LINKED ISSUE #46: FRONTEND | Login Form Component

### PR DESCRIPTION
**Pull request by vieraboschkova**
close #46

### CODE REVIEW

File: client/src/domain/Auth/Login/index.js [hunk 1]
```diff
@@ -1,7 +1,64 @@
...
-  return <div className="login">Rendering Login in here</div>;
+  const [formData, setFormData] = useState({
+    email: '',
+    password: '',
+  });
+
+  const { email, password } = formData;
+
+  const onChange = (event) => {
+    setFormData({ ...formData, [event.target.name]: event.target.value });
```
Comment (aleksanderwalczuk) [inline client/src/domain/Auth/Login/index.js:14]: good job 👍

File: client/src/domain/Auth/Login/index.js [hunk 2]
```diff
@@ -1,7 +1,64 @@
...
+            value={password}
+            onChange={onChange}
+            minLength="5"
+          />
+        </div>
+        {/* TODO: add submit logic */}
+        <button type="submit" className="button--submit">
+          Login
+        </button>
+      </form>
```
Comment (aleksanderwalczuk) [inline client/src/domain/Auth/Login/index.js:56]: Consider using a separate package for controlled inputs. E.g. Formik or React useForm Hook. You can also use yup for validation schemas.

**Review by aleksanderwalczuk — APPROVED** [review]
Another npm package might be useful 😃

File: client/src/domain/Auth/Login/index.js (hunk 2 above)
Comment (vieraboschkova) [reply client/src/domain/Auth/Login/index.js:56]: Thanks, I'll check it out!

**Pull request merged**